modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -9028,6 +9028,59 @@ public void testPatchScopeScopeKeyByExternalReferenceCode()
 			group.getGroupKey());
 	}
 
+	@Test
+	public void testPostByExternalReferenceCodeComment() throws Exception {
+
+		// Company scope
+
+		ObjectEntry companyObjectEntry = ObjectEntryTestUtil.addObjectEntry(
+			_objectDefinition1, _OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1);
+
+		String comment1 = RandomTestUtil.randomString();
+
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
+			JSONUtil.put(
+				"externalReferenceCode", _ERC_VALUE_1
+			).put(
+				"text", comment1
+			).toString(),
+			StringBundler.concat(
+				_getEndpoint(_objectDefinition1, 0),
+				"/by-external-reference-code/",
+				companyObjectEntry.getExternalReferenceCode(), "/comments"),
+			Http.Method.POST);
+
+		Assert.assertEquals("<p>" + comment1 + "</p>", jsonObject.get("text"));
+		Assert.assertEquals(
+			_ERC_VALUE_1, jsonObject.get("externalReferenceCode"));
+
+		// Site scope
+
+		Group group = _groupLocalService.fetchGroup(_testGroupId);
+
+		ObjectEntry siteObjectEntry = ObjectEntryTestUtil.addObjectEntry(
+			_siteScopedObjectDefinition1, _OBJECT_FIELD_NAME_1,
+			_OBJECT_FIELD_VALUE_1);
+
+		String comment2 = RandomTestUtil.randomString();
```

Comment:
Is that possible to reuse the variable `comment` to avoid having `comment1` and `comment2?` Like you did for `jsonObject`?